### ISSUE TITLE
`major.push` is not a function

### ISSUE THREAD

**Issue by Kocal** [browserslist/browserslist-useragent-regexp#409]
Hi,

I'm facing an issue when running `yarn browserslist-useragent-regexp --allowHigherVersions`:

```log
➜  ALP git:(fix/YP-5389) ✗ yarn browserslist-useragent-regexp --allowHigherVersions
yarn run v1.21.1
$ /home/kocal/workspace/ALP/node_modules/.bin/browserslist-useragent-regexp --allowHigherVersions
/home/kocal/workspace/ALP/node_modules/browserslist-useragent-regexp/lib/index.js:1174
          major.push(current[exports.SemverPart.Major]);
                ^

TypeError: major.push is not a function
    at versionsListToRanges (/home/kocal/workspace/ALP/node_modules/browserslist-useragent-regexp/lib/index.js:1174:17)
    at browsers.forEach (/home/kocal/workspace/ALP/node_modules/browserslist-useragent-regexp/lib/index.js:1216:24)
    at Map.forEach (<anonymous>)
    at browserVersionsToRanges (/home/kocal/workspace/ALP/node_modules/browserslist-useragent-regexp/lib/index.js:1215:12)
    at getUserAgentRegExps (/home/kocal/workspace/ALP/node_modules/browserslist-useragent-regexp/lib/index.js:1515:26)
    at Object.getUserAgentRegExp (/home/kocal/workspace/ALP/node_modules/browserslist-useragent-regexp/lib/index.js:1529:19)
    at Object.<anonymous> (/home/kocal/workspace/ALP/node_modules/browserslist-useragent-regexp/lib/cli.js:133:17)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
error Command failed with exit code 1.
```

I'm using:
- node 10.18.0
- yarn 1.21.1
- browsersist-useragent-regexp last version (2.0.1)
- and my browserslist configuration in `package.json`:
```json
  "browserslist": [
    "> 0.25%",
    "not ie 11",
    "not op_mini all"
  ],
```

Thank you!

**Comment by dangreen** [comment]
Fixed in [v2.0.2](https://github.com/browserslist/browserslist-useragent-regexp/releases/tag/v2.0.2)

**Comment by Kocal** [comment]
Thank you, I will give it a try!
